src/mscorlib/shared/System/Memory.cs

```diff
@@ -120,20 +120,17 @@ public Memory(T[] array, int start, int length)
         /// <param name="manager">The memory manager.</param>
         /// <param name="start">The index at which to begin the memory.</param>
         /// <param name="length">The number of items in the memory.</param>
-        /// <remarks>Returns default when <paramref name="manager"/> is null.</remarks>
+        /// <exception cref="System.ArgumentNullException">
+        /// Thrown when <paramref name="manager"/> is null.
+        /// </exception>
         /// <exception cref="System.ArgumentOutOfRangeException">
         /// Thrown when the specified <paramref name="start"/> or end index is not in the range (&lt;0 or &gt;=Length).
         /// </exception>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public Memory(MemoryManager<T> manager, int start, int length)
         {
             if (manager == null)
-            {
-                if (start != 0 || length != 0)
```

Comment:
This is a breaking change. It is ok only if this gets ported to 2.1. Once we ship this in 2.1, it needs to stay this way.

Reply:
My understanding is that everything in master until at least some time next week will end up in 2.1.

Reply:
Yes there will be a last master->release/2.1 integration on April 13 for RC.  After which the master branch becomes "2.2"